lib/tasks/analyse/links.rb

```diff
@@ -75,26 +76,31 @@ def write_taxon_worksheet(results, spreadsheet)
       worksheet: worksheet,
     )
 
-    results_by_taxon_base_path = group_results(results, :taxon_base_path)
-
     row_number = 1
-    results_by_taxon_base_path.each_pair do |taxon_base_path, base_path_results|
-      navigation_page_type = base_path_results[0][:navigation_page_type]
 
-      if navigation_page_type == 'accordion'
-        base_path_results += base_path_results.map do |result|
-          result.merge(section: 'all')
+    results_by_navigation_page_type = group_results(results, :navigation_page_type)
```

Comment:
Hi,

I think I will leave it weighing up how much we will reuse this and time it will take.